src/musicbrainz/chromaprinter.cpp

```diff
@@ -12,6 +12,16 @@
 
 namespace
 {
+// Type declarations of *fprint and *encoded pointers need to account for Chromaprint API version
+// (void* -> uint32_t*) and (void* -> char*) changed in versions v1.4.0 or later -- alyptik 12/2016
+#if (CHROMAPRINT_VERSION_MINOR > 3) || (CHROMAPRINT_VERSION_MAJOR > 1)
+    typedef uint32_t* uint32_p;
+    typedef char* char_p;
```

Comment:
Well, my fault, I was abused by github way of presenting commit parts.
It was fine. sorry about that.

The way it is now, with the lase commit, is fine for me too, I think this is no problem if #if is not at the beginning of line as it is more readable inside namespace.